src/sentry/static/sentry/app/views/stream/stream.jsx

```diff
@@ -647,6 +656,7 @@ const Stream = createReactClass({
           projectId={projectId}
           statsPeriod={statsPeriod}
           query={this.state.query}
+          hasGuideAnchor={hasGuideAnchor}
```

Comment:
It should be sufficient to ping the first issue only. Pinging all issues adds a lot of noise to the page and doesn't add any value :)